lib/helpers/validateRegistry.js

```diff
@@ -2,17 +2,15 @@
 
 var assert = require('assert');
 
-var _ = require('lodash');
-
 function isConstructor(registry) {
   if (!(registry && registry.prototype)) {
     return false;
   }
 
-  var hasProtoGet = _.isFunction(registry.prototype.get);
-  var hasProtoSet = _.isFunction(registry.prototype.set);
-  var hasProtoInit = _.isFunction(registry.prototype.init);
-  var hasProtoTasks = _.isFunction(registry.prototype.tasks);
+  var hasProtoGet = typeof registry.prototype.get === 'function';
```

Comment:
probably make this a helper function local to this file.  these changes add a lot of noise to the implementation